src/riak_ensemble_peer.erl

```diff
@@ -30,6 +30,7 @@
 -export([join/2, join/3, update_members/3, get_leader/1, backend_pong/1]).
 -export([kget/4, kget/5, kupdate/6, kput_once/5, kover/5, kmodify/6, kdelete/4,
          ksafe_delete/5, obj_value/2, obj_value/3]).
+-export([debug_local_get/2]).
```

Comment:
Any possibility of only exporting this when TEST is true? Always hate exposing things (even somewhat-clearly-named things) that normal users shouldn't use if we can help it.